### DISCUSSION TITLE
Extract impedance values form "2D Extractor: stripline analysis"

### DISCUSSION THREAD

**Discussion by giuseppeselli** [ansys/pyaedt#3887] [General]
greetings, I am following the example: "2D Extractor: stripline analysis". At the end it plots the differential impedance, but I would like to output the values, (print them out), but I am having  hard time locating them within the variable **a**. Any suggestions are greatly appracieted.

**Comment by svandenb-dev** [comment]
@giuseppeselli OK i see you need help on get_solution_data(). We will come back to you shortly.

**Comment by Samuelopez-ansys** [comment]
Hi @giuseppeselli,

If you debug the code, you will figure out how the get_solution_data object works. This method returns an object called SolutionData, which is very powerful.

<img width="461" alt="image" src="https://github.com/ansys/pyaedt/assets/85613111/191f891d-4d67-4c81-aaac-343cbb678ac9">

YOu can manipulate all data inside "a"., for example, a.data_magnitude() gives you the values in magnitude, you can do dir(a) and you will discover all the available methods and properties (like any kind of python object). You can find in the Examples many different cases of post-processing manipulation:

https://aedt.docs.pyansys.com/version/stable/examples/05-Q3D/Q3D_Example.html#solve-setup

https://aedt.docs.pyansys.com/version/stable/examples/03-Maxwell/Maxwell2D_NissanLeaf.html#get-solution-data
